asreview/project.py

```diff
@@ -132,46 +94,24 @@ def decorated_function(project_id, *args, **kwargs):
     return decorate
 
 
-# !=============================================
-# @TODO: Another thing that needs consideration:
-# I need ownership id's to avoid collaborators inviting
-# new collaborators. That means I'd like to alter
-# the ASReviewProject class -> adding an ownership
-# attribute. I could add it on the class itself, but
-# this is only needed in the authenticated version.
-# Instead I will add it dynamically in this function...
-# Feedback is required.
-# !=============================================
-def list_asreview_projects(user=None):
```

Comment:
I think we should deprecate this one. This is public API and there can be users. I like the rename btw.

Reply:
You mean make it a private function, or remove it altogether? Right now `get_projects` is used only in `webapp/api/projects.py` (2 times) and `webapp/start_flask.py`. It would be easy enough to completely remove it. Though I guess it could be useful so making it private would make some sense.